pvlib/test/test_pvsystem.py

```diff
@@ -553,43 +553,112 @@ def test_v_from_i(fixture_v_from_i):
     assert_allclose(V, V_expected, atol=atol)
 
 
+@requires_scipy
+def test_v_from_i_brentq(fixture_v_from_i):
```

Comment:
I believe you can remove the second test if you do this:

```python
@requires_scipy
@pytest.mark.parametrize('method,atol', [('brentq', 1e-11), ('newton', 1e-8)])
def test_v_from_i_methods(fixture_v_from_i, method, atol):
# remove atol spec and set method=method
```

Reply:
yep, i'm feeling very tired, I will add a folow up pr to clean thew of,oo?